build.gradle

```diff
@@ -307,7 +307,12 @@ def createCheckTypeTask(projectName, checker, shortName, args = []) {
                 '-Xmaxerrs', '10000',
                 '-Xmaxwarns', '10000',
                 '-ArequirePrefixInWarningSuppressions',
-                ]
+        ]
+        if (shortName.equals("Interning")) {
```

Comment:
This should be passed when creating the `checkIntering` class:
`createCheckTypeTask(project.name, 'org.checkerframework.checker.interning.InterningChecker', 'Interning', ['-Astubs=javax-lang-model-element-name.astub'])`